starlite/config/__init__.py

```diff
@@ -1,7 +1,10 @@
+from starlite.logging import QueueListenerHandler
```

Comment:
Should this also be exported here?  it looks like it's still in the original `starlite.logging.__init__`

Reply:
I didn't move it to starlite.config because it's a function of the logging itself and unrelated to configuration

Reply:
Wait, I think I see what you mean.  It doesn't need to be exported out of config.\_\_init\_\_.  I'll make a change